test/e2e/framework/test_context.go

```diff
@@ -73,7 +73,7 @@ type TestContextType struct {
 	// only master Node will be monitored.
 	GatherKubeSystemResourceUsageData string
 	GatherLogsSizes                   bool
-	GatherMetricsAfterTest            bool
+	GatherMetricsAfterTest            string
```

Comment:
It shouldn't.. As all checked that all jobs on our CI either don't set the flag at all or set it as `--gather-metrics-at-teardown=true` (basically just our scalability tests)